protos/followme/followme.proto

```diff
@@ -0,0 +1,120 @@
+syntax = "proto3";
+
+package mavsdk.rpc.followme;
+
+option java_package = "io.mavsdk.followme";
+option java_outer_classname = "FollowMeProto";
+
+service FollowMeService {
+  /*
+   * Gets current FollowMe configuration.
```

Comment:
Nitpick: because they all fit in one line, could we use `//` instead? Like [here](https://github.com/mavlink/MAVSDK-Proto/blob/master/protos/telemetry/telemetry.proto#L13) :innocent: